### PR TITLE
Blocking edges to 4.5.1

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: blocked-edges/4.5.1.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+to: 4.5.1
+from: *
```
Comment (sdodson) [inline blocked-edges/4.5.1.yaml:2]: All the other examples are `from: .*` i don't think this is a glob.

Comment (sdodson) [reply blocked-edges/4.5.1.yaml:2]: ex https://github.com/openshift/cincinnati-graph-data/blob/master/blocked-edges/4.1.1.yaml

Comment (LalatenduMohanty) [reply blocked-edges/4.5.1.yaml]: Good catch, thanks

**Comment by sdodson** [comment]
@eparis @jwforres you're up for approval

**Pull request merged**